plugin/src/main/java/org/jenkinsci/plugins/structs/describable/DescribableModel.java

```diff
@@ -255,9 +259,23 @@ public String getDisplayName() {
         return type.getSimpleName();
     }
 
+    @Deprecated
```

Comment:
Polite to add a `@deprecated` Javadoc tag which `@link`s to the now-recommended overload.